### ISSUE TITLE
Cobbler do not support UEFI booting mode in ISO image

### ISSUE THREAD

**Issue by ujnzxw** [cobbler/cobbler#1843]
Hi Cobbler Team,

`cobbler buildiso` is a great feature for building customization ISO image.

Currently, I found that the exported ISO image only support legacy BIOS booting mode but do not support the latest UEFI boot mode. You know UEFI is the default option in some hardware like HP DL380. 

Does your team have a plan to support this feature?  

Thanks,
Steven

**Comment by rpasche** [comment]
@jmaas 

Consider using ```grub2-mkrescue``` to create a UEFI bootable ISO image. I made an - not that pretty - patch to ```action_buildiso.py``` today on a test system. Currently, it just creates a ```grub.cfg``` file in parallel to the ```isolinux.cfg``` file. After the ISO has been created, I - currently - run ```grub2-mkrescue -o <iso> /var/cache/cobbler/buildiso``` to use the ```buildisodir``` as source for ```grub2-mkrescue```.

This ISO is bootable from UEFI systems. I'll create another PR for this, which you might want to use as template.

**Comment by jmaas** [comment]
Great! Looking forward to your PR, this has been on the TODO list for way
too long.
Glad someone is pitching in! :)

**Comment by SchoolGuy** [comment]
So we have the older issue #1777 so I will close this as a duplicate one and track my progress there. I am not closing this because I won't do it. Please move any comments or suggestions over to #1777!
